test/elixir_console_web/features/console_test.exs

```diff
@@ -41,4 +42,24 @@ defmodule ElixirConsoleWeb.ConsoleTest do
       |> assert
     end)
   end
+
+  feature "visitor can get official documentation of an elixir function", %{session: session} do
+    session
+    |> visit("/")
+    |> fill_in(@command_input, with: "String.length('elixir')")
+    |> send_keys([:enter])
+    |> find(@command_output, fn output ->
+      output
+      |> assert_text("> String.length('elixir')")
+      |> click(css("#command0 > span"))
```

Comment:
Wondering whether we could make this line a tiny bit more readable/understandable, in the sense that's attempting to click on the function link to documentation.

Thoughts on doing something like 

```diff
diff --git a/lib/elixir_console_web/live/console_live/history_component.ex b/lib/elixir_console_web/live/console_live/history_component.ex
index 8e1205f..dc89241 100644
--- a/lib/elixir_console_web/live/console_live/history_component.ex
+++ b/lib/elixir_console_web/live/console_live/history_component.ex
@@ -45,7 +45,7 @@ defmodule ElixirConsoleWeb.ConsoleLive.HistoryComponent do
          docs: docs,
          link: link
        }) do
-    ~e{<span
+    ~e{<a href="#"
     phx-click="function_link_clicked"
     phx-value-func_name="<%= func_name %>"
     phx-value-header="<%= header %>"
@@ -53,7 +53,7 @@ defmodule ElixirConsoleWeb.ConsoleLive.HistoryComponent do
     phx-value-link="<%= link %>"
     phx-target="#commandOutput"
     class="<%= inline_help_class_for(function_or_operator) %>"
-    ><%= part %></span>}
+    ><%= part %></a>}
   end
 
   defp inline_help_class_for(:function), do: "text-green-400 cursor-pointer underline"
diff --git a/test/elixir_console_web/features/console_test.exs b/test/elixir_console_web/features/console_test.exs
index 2f83316..462025d 100644
--- a/test/elixir_console_web/features/console_test.exs
+++ b/test/elixir_console_web/features/console_test.exs
@@ -2,7 +2,7 @@ defmodule ElixirConsoleWeb.ConsoleTest do
   use ExUnit.Case, async: true
   use Wallaby.Feature
 
-  import Wallaby.Query, only: [css: 1]
+  import Wallaby.Query, only: [css: 1, link: 1]
 
   @command_input css("#commandInput")
   @command_output css("#commandOutput")
@@ -85,7 +85,7 @@ defmodule ElixirConsoleWeb.ConsoleTest do
     |> find(@command_output, fn output ->
       output
       |> assert_text("> String.length('elixir')")
-      |> click(css("#command0 > span"))
+      |> click(link("String.length"))
     end)
     |> find(@documentation_output, fn documentation_output ->
       assert_text(documentation_output, "String.length/1")
```

?

If we prefer keep it as a span, maybe we just add a code comment in the test case explaining what the click is intended to click on the UI for the sake of better "understandability"?

Reply:
cc/ @eldano

Reply:
The only difference I see is that on hover the browser (at least Chrome) shows in the bottom corner the url you'd be redirected by following the link. If it's a span it doesn't show anything.

Another option is to find the span by one if its attributes which is a bit more clear where you're clicking, although the css is a bit uglier:
```elixir
...
|> click(css("[phx-value-func_name='String.length/1']"))
```

Reply:
> The only difference I see is that on hover the browser (at least Chrome) shows in the bottom corner the url you'd be redirected by following the link. If it's a span it doesn't show anything.

Ah, right... don't like that he.

> 
> Another option is to find the span by one if its attributes which is a bit more clear where you're clicking, although the css is a bit uglier:
> 
> ```elixir
> ...
> |> click(css("[phx-value-func_name='String.length/1']"))
> ```

Yeah...

Not ideal given it's still a bit to coupled with the implementation like the `#command0 span`, but I think it's a good compromise for this particular situation.

I'd go with this last option for now :-) :+1: 

cc/ @matiaswilner.

Reply:
Thanks for the update!